ui/src/Components/Dashboard/Table/TableHeader/SortCursor.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+
+import { TABLE } from "../../../../Constant/constant";
+
+export default function SortCursor(props) {
+  const [currentClickedCursor, setCurrentClickedCursor] = useState("");
+
+  const onClickUpArrow = (args, headerName) => {
```

Comment:
Suggestion: Merge this to one method and on click pass the TYPE (is it up or down arrow) and accordingly pass the arguments. This could move to utils then.

Reply:
@Neha yes makes sense to merge both functions into one wil do that. But do we want the onClick function to be in utils?

Reply:
I think we can keep the function in the component itself...as I don't think so, we have anything which could be generic. Thoughts?

Reply:
Yes nothing generic. I would keep it here.